README.md

```diff
@@ -7,7 +7,33 @@ By default, each TeamCity installation runs under a Professional Server license
 
 By pulling this image you accept the [JetBrains license agreement for TeamCity (Commercial License)] (https://www.jetbrains.com/teamcity/buy/license.html)
 
-This docker image can use (optionally) an external postgress database instead of the internal database of teamcity. (recommended for production usage)
+This docker image can use (optionally) an external postgress or mysql database instead of the internal database of TeamCity. (recommended for production usage)
+
+How to use this image?
+---------------
+
+A typical way to create the container and make sure it gets restarted:
+
+    docker run -v /var/lib/teamcity:/var/lib/teamcity -v /dev/urandom:/dev/random -p 8111:8111 --restart=always --name teamcity teamcity
+
+Note the `urandom` volume. Without it TeamCity can take up to 30 minutes to start due to waiting for bits to appear in `/dev/random`. (Don't worry, [it's okay](http://www.2uo.de/myths-about-urandom/).)
```

Comment:
This is not a teamcity or docker issue. You probally have an low-entropy system (e.g. virtual). I have not experienced this since we run it on a dedicated server, maybe add that as a comment that it is not always needed?